bin/prompt.js

```diff
@@ -0,0 +1,15 @@
+const inquirer = require('inquirer')
+
+const projects = require('../lib/projects')
+
+module.exports = async function () {
+  let a = await inquirer.prompt([
+    {
+      type: 'list',
+      name: 'project',
+      message: 'Select one of the following projects.',
+      choices: Object.keys(projects)
```

Comment:
This definitely makes it more complicated, but would it be possible to use the project names as defined in `projects.json` and include the description? That would be an A++ enhancement to the experience 🤗

Full disclosure, I was working on doing this and got stuck 😅

Reply:
i dont find a better way of doing this as of now

Reply:
Let's create an issue 👍